chaco/scatter_inspector_overlay.py

```diff
@@ -41,7 +42,7 @@ class ScatterInspectorOverlay(AbstractOverlay):
     # using a series of trait change handlers (defined at the end of the
     # class)
```

Comment:
This is tangential but i want to dig a little and find the source of this comment - it looks like there was going to be some fancy traits feature which would remove unnecessary code.